src/pages/dormitory/call_names.tsx

```diff
@@ -28,74 +36,138 @@ function rgba2rgbOnWhite(
   return [_red, _green, _blue]
 }
 
-export default function CallNamesPage() {
+function Column({
```

Comment:
横一列は多分Rowのはず･･･！

Reply:
なぜかプッシュができなかったのでとりあえずコメントを！
```suggestion
// FIXME: Row
function Column({
```

Reply:
本当ですね Row ですね……
(Flutter で縦並びするのが Column だったのでてっきり行だと思ってました)

そして push できなかったですか (Allow edits and access to ... のチェックは外してなかったはずだが……).  
またの機会があれば FIXME 解消しておきますね！